dev-util/xmake/Manifest

```diff
@@ -0,0 +1,6 @@
+DIST xmake-2.5.3-ext_core-lua-cjson.tar.gz 67483 BLAKE2B 4baaaaff0c6bdb0c69798377431f7ef00a9b3ffd9f14d8e792c12d4bad9ad7c941e614a06afa149da5becf101ffda8d09b41599f5a66100309e39c210d8f87b3 SHA512 775f67c653970c9e0f58dbf14c3c865cb91d90c17d3cc2438afa9a5e9d55941b21f966b3a276a57b2fc528390b74c8b17b9ca13c497cbd945bedb4fdb371aff7
+DIST xmake-2.5.3-ext_core-luajit.tar.gz 1037801 BLAKE2B 6b9ec0022dc78e51b03dd492cbf4376801c27569286285e5054019a51ed0916cfdf8e035f188f233c47f587591b3676692c32d4040378c7a792dfb92af70f144 SHA512 58c0f7889604b4a54d31e31727fe388a01e9f9cdec3e18d16c6f394724de246776d77db8cb8b131b132fd3ee9e962d4fd4aa6fab889d7a764f6d41820ab08623
+DIST xmake-2.5.3-ext_core-sv.tar.gz 25882 BLAKE2B 424fe1243dd276a2fe0a394cd1171aad25ca3092dc299d47bade9dc3d609caf76bfda7c84248a1036c6dab796dfdf80913a570a1540bfe30781f46411594042f SHA512 95267648128bb7ed41f91151f91b28345c5a19ed49b67a4d88d2a487eda4058a8cb55e42b76bad86532242e1f72f20fb35e1dd5379653bc26822f270580b5308
+DIST xmake-2.5.3-ext_tboox-tboox.tar.gz 807351 BLAKE2B 34d62cc97ec7f22722a3737b806e15047f98a57105c1e2419a95dae17cd12a1b7724ed006dcd3485ccf8c3d8600f8fd26bdb16e557524001bd97dca4dbd7b19c SHA512 2c92fbdfcf04c741fa4c8bd68baef2e5524f44bc4fec72c938a63d403b1b062d53ec523606a7e425adbbcacd5b16742791d7fb6b59674d179e0fd741cf8fd1ba
+DIST xmake-2.5.3.tar.gz 2077299 BLAKE2B 24863d4a156992809bfc9e2e9222230cb75ef034cfdef3828e1ec1f746fbea0d3ca88e4d249762e9aca2bde5361bc9b407e323491cbbdc05d0a52b5f88c542c1 SHA512 3a4bc6df44850259fe262b9ca8435eb46ad5a8b70bf8e715782a524fc70949ae6343923dba0991aef07e2ddcb9814b650b619fd6390dcb7f9454f667a5e7fd2d
```

Comment:
https://github.com/xmake-io/xmake/releases/download/v2.5.3/xmake-v2.5.3.tar.gz already contains all the submodules source code, you can use it directly, only a tar package is needed .

Reply:
I don't think so. The needed directories are completely empty. That was the first tarball I checked.

Reply:
Wait no, I'm stupid. That tarball I saw I wasn't exactly sure what I was supposed to do with it.

Reply:
I can be sure that it has all the submodules and makefiles, because I am using it myself. The homebrew package is also made with it.

Reply:
`tar xf xmake-v2.5.3.tar.gz`
`cd xmake-v2.5.3.tar.gz`
`fd makefile`:
```
plugins/project/make/makefile.lua
plugins/project/make/xmakefile.lua
```
And the root of the archive (`ls -a1`) looks like:
```
.
..
actions
core
includes
languages
modules
platforms
plugins
repository
rules
scripts
templates
themes
toolchains
```
It looks like the contents of `xmake/` in the the xmake repo.

Reply:
Is [this](https://github.com/xmake-io/xmake/blob/master/scripts/brew-pr) what you were talking about?
It looks like it uses the `v2.3.1` tarball.

Reply:
Are you sure use releases/xmake-v2.5.3.tar.gz instead of archve tar?

you can see https://github.com/Homebrew/linuxbrew-core/blob/master/Formula/xmake.rb

```console
$ tar -xvf ./xmake-v2.5.3.tar.gz 
$ find ./ -name makefile
.//core/makefile
.//core/src/demo/makefile
.//core/src/sv/makefile
.//core/src/lcurses/makefile
.//core/src/makefile
.//core/src/lua-cjson/makefile
.//core/src/tbox/makefile
.//core/src/xmake/makefile
.//core/src/luajit/makefile
.//makefile
```

Reply:
> tar xf xmake-v2.5.3.tar.gz
cd xmake

Please do not `cd xmake`.

Reply:
OH that's why. There is no containing folder for everything.
You should place everything in a dedicated folder, 
and when that folder is unnecessary the user can append `--strip-components 1` to not have it.

Tarballs without a containing folder is called tarbombing and that leads to both confusion and clutter.

Reply:
Anyways, now that I know I'll update the ebuild.

Reply:
Ok, I have merged it, thanks. :+1: